outline/manager.js

```diff
@@ -365,19 +368,73 @@ module.exports = function (doc) {
   }
   this.getDashboardItems = getDashboardItems
 
-  async function showDashboard (container, unselectCurrentPane, globalPaneToSelect) {
-    container.innerHTML = ''
-    // console.log(container)
-    const currentPane = dom.querySelector('#outline .paneShown')
-    if (unselectCurrentPane && currentPane) {
-      // eslint-disable-next-line no-undef
-      // currentPane.dispatchEvent(new Event('clglobalAppTabsick'))
+  /**
+   *
+   * @param {HTMLElement} container The element that the dashboard should append dashboard to
+   * @param {Object} [options] A set of options that can be passed
+   * @param {string} [options.pane] To open a specific dashboard pane
+   * @returns {Promise<void>}
+   */
+  async function showDashboard (options = {}) {
+    const dashboardContainer = getDashboardContainer()
+    const outlineContainer = getOutlineContainer()
+    // reuse dashboard if already children already is inserted
+    if (dashboardContainer.childNodes.length > 0 && options.pane) {
+      outlineContainer.style.display = 'none'
+      dashboardContainer.style.display = 'inherit'
+      const tab = dashboardContainer.querySelector(`[data-name="${options.pane}"]`) ||
+        dashboardContainer.querySelector('[data-name]')
```

Comment:
I presume `data-name` is already used somewhere? If not, it's a bit odd to have a data attribute with the same name as a regular attribute (i.e. `name`), and something like `data-global-pane-name` or something would be clear. But no biggy if it's already used.

Reply:
You're correct, I should use a more specific name than one that resembles standard attribute `name`. Will use `data-global-pane-name` instead.